linux/network_device_ops/enable_new_ethernet.yml

```diff
@@ -69,19 +69,27 @@
     - guest_os_with_gui is defined and not guest_os_with_gui
 
 # Bring up the new network interface
+- name: "Bring link '{{ eth_dev }}' up"
+  shell: "nmcli device connect {{ eth_dev }}"
+  delegate_to: "{{ vm_guest_ip }}"
+  register: set_link_down_up
+  when:
+    - guest_os_family == "RedHat"
+    - guest_os_ansible_distribution_major_ver | int >= 8
+
 - name: "Bring link '{{ eth_dev }}' up"
   shell: "ip link set {{ eth_dev }} down; ip link set {{ eth_dev }} up"
   delegate_to: "{{ vm_guest_ip }}"
   register: set_link_down_up
-  when: >
-    (guest_os_ansible_distribution in ["VMware Photon OS", "Flatcar"]) or
-    (guest_os_ansible_distribution == "Ubuntu" and guest_os_with_gui is defined and guest_os_with_gui)
+  when: guest_os_ansible_distribution in ["VMware Photon OS", "Flatcar", "Ubuntu"]
```

Comment:
Is this the same in Ubuntu with desktop and without desktop?
```
(guest_os_ansible_distribution == "Ubuntu" and guest_os_with_gui is defined and guest_os_with_gui)
```

Reply:
Ubuntu without desktop was not dealt with in previous workflow. I haven't run a full regression yet, will double check with Ubuntu Desktop and Live server.

Reply:
Ubuntu server network config is above this block, so remove changes about ubuntu